src/invocation/InvocationService.ts

```diff
@@ -162,3 +170,44 @@ export class InvocationService {
     }
 }
 
+export class ListenerService {
```

Comment:
Do you mean using ListenerService? It is used in https://github.com/hazelcast/hazelcast-nodejs-client/pull/68/files#diff-5371374fe67ebe5287b4903ca93e33ceR73 . It will be used a lot more in newer releases.